tests/test-sources/plugins/by-name/no-neck-pain/default.nix

```diff
@@ -0,0 +1,11 @@
+{
+  empty = {
+    plugins.no-neck-pain.enable = true;
+  };
+
+  defaults = {
+    plugins.no-neck-pain = {
+      enable = true;
```

Comment:
Should add at least some of the defaults from https://github.com/shortcuts/no-neck-pain.nvim/blob/main/lua/no-neck-pain/config.lua#L148-L338 for testing the plugin.